examples/elasticsearch/src/buildRequest.js

```diff
@@ -0,0 +1,121 @@
+import buildRequestFilter from "./buildRequestFilter";
+
+function buildFrom(current, resultsPerPage) {
+  if (!current || !resultsPerPage) return;
+  return (current - 1) * resultsPerPage;
+}
+
+function buildSort(sortDirection, sortField) {
+  if (sortDirection && sortField) {
+    return [{ [`${sortField}.keyword`]: sortDirection }];
+  }
+}
+
+function buildMatch(searchTerm) {
+  return searchTerm
+    ? {
+        multi_match: {
+          query: searchTerm,
+          fields: ["title", "description"]
+        }
+      }
+    : { match_all: {} };
+}
+
+/*
+
+  Converts Current Application State to an Elasticsearch request.
+
+  When implementing an onSearch Handler in Search UI, the handler needs to take the
+  current state of the application and convert it to an API request.
+
+  For instance, there is a "current" property in the application state that you receive
+  in this handler. The "current" property represents the current page in pagination.
+
+  There is no concepts of "pages" in Elasticsearch, so the closest thing we have to this is
+  the "from" option. This file take the "current" value (the current page), and the "resultsPerPage"
+  value (the size of a page), to calculate the "from" offset for Elasticsearch.
```

Comment:
I don't reallyyy understand what this is trying to say. We might need to unpack this more and think of a different way to describe the "from" offset.